### PR TITLE
Fix: Hover over tabs is not working as expected in Nested Tabs [ed-8845]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 1]
```diff
@@ -170,8 +155,10 @@
 			}
 
 			&.e-active {
-				color: var( --n-tabs-title-color-active );
-				background-color: var( --n-tabs-title-background-color-active );
+				&:not(:hover) {
+					color: var( --n-tabs-title-color-active );
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:159]: I needed this to test the active state.

I have decided not to change the active tab styling on hover.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 2]
```diff
@@ -99,36 +99,22 @@
...
+				transition: all var( --n-tabs-title-transition );
+			}
+
 			&-text {
 				display: flex;
 			  	align-items: center;
 			  	font-size: var( --n-tabs-title-font-size );
 			}
 
-			&:hover {
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:112]: I have moved the &:hover styling downwards.

Comment (davseve) [reply modules/nested-tabs/assets/scss/frontend.scss:112]: I see that `--n-tabs-title-background-color-hover` has no assignment.

**Review by davseve — DISMISSED** [review]

File: modules/nested-tabs/assets/scss/frontend.scss (hunk 2 above)
Comment (hein-obox) [reply modules/nested-tabs/assets/scss/frontend.scss:112]: Can I delete it? Is it managed by the php?
Or should we set the default value?

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**